crawler/lookingglass_crawler/spiders/media_spider.py

```
@@ -0,0 +1,165 @@
+                self.bucket,
+                obj_name,
+                io.BytesIO(html_bytes),
+                length=len(html_bytes),
+                part_size=10 * 1024 * 1024,
+                content_type="text/html",
+                metadata=metadata,
+            )
+        except Exception as e:
+            self.logger.error(f"❌ Failed uploading page {url}: {e}")
+        finally:
+            return []  # No items
```

Ruff (0.11.9)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)